src/main/java/com/google/devtools/build/lib/rules/filegroup/Filegroup.java

```diff
@@ -75,7 +76,8 @@ public ConfiguredTarget create(RuleContext ruleContext)
         InstrumentedFilesCollector.collect(ruleContext,
             // what do *we* know about whether this is a source file or not
             new InstrumentationSpec(FileTypeSet.ANY_FILE, "srcs", "deps", "data"),
-            InstrumentedFilesCollector.NO_METADATA_COLLECTOR, filesToBuild);
+            InstrumentedFilesCollector.NO_METADATA_COLLECTOR, filesToBuild,
```

Comment:
Nit: each argument on a separate line, including 1st arg, the rule context, that is  misformatted@HEAD, thanks!